### DISCUSSION TITLE
Is there a way to access the separation mask?

### DISCUSSION THREAD

**Discussion by hananbo26** [asteroid-team/asteroid#507] [Q&A]
When I am using a pretrained model to infer. 
Is there a way to access the separation mask in the STFT domain?

**Comment by mpariente** [comment]
Have a look at the functions in `BaseModel.forward`, there are function calls and it's pretty clear where is the mask.

**Reply by hananbo26** [reply]
Thanks for your response🙏

I succeeded to access the TF masks estimated by a pretrained model but to the best of my understanding these masks are not in the STFT domain but in some other TF representation. 

I need the mask in the STFT domain.

Is there a pretrained model for speech separation that uses the STFT as encoder? or any other way to access the STFT masks?

**Reply by mpariente** [reply]
Great!
I don't think we have any STFT-based pretrained model for now, sorry. 
If you do train one, please share it :wink:
